layersvt/screenshot.cpp

```diff
@@ -1207,16 +1235,16 @@ VKAPI_ATTR VkResult VKAPI_CALL QueuePresentKHR(VkQueue queue, const VkPresentInf
 
             if (screenshotFrames.empty() && isEndOfScreenShotFrameRange(frameNumber, &screenShotFrameRange)) {
                 // Free all our maps since we are done with them.
-                for (auto it = swapchainMap.begin(); it != swapchainMap.end(); it++) {
-                    SwapchainMapStruct *swapchainMapElem = it->second;
+                for (auto swapchainIter = swapchainMap.begin(); swapchainIter != swapchainMap.end(); swapchainIter++) {
+                    SwapchainMapStruct *swapchainMapElem = swapchainIter->second;
                     delete swapchainMapElem;
                 }
-                for (auto it = imageMap.begin(); it != imageMap.end(); it++) {
-                    ImageMapStruct *imageMapElem = it->second;
+                for (auto imageIter = imageMap.begin(); imageIter != imageMap.end(); imageIter++) {
+                    ImageMapStruct *imageMapElem = imageIter->second;
                     delete imageMapElem;
                 }
-                for (auto it = physDeviceMap.begin(); it != physDeviceMap.end(); it++) {
-                    PhysDeviceMapStruct *physDeviceMapElem = it->second;
+                for (auto physDeviceIter = physDeviceMap.begin(); physDeviceIter != physDeviceMap.end(); physDeviceIter++) {
+                    PhysDeviceMapStruct *physDeviceMapElem = physDeviceIter->second;
```

Comment:
Right it's unrelated to this change, but I saw a build warning related to the duplicate used variable name (there's already an iterator called 'it', so changed this one to ensure there wasn't any conflicts.